app/views/brexit_checker/email_signup.html.erb

```diff
@@ -28,15 +28,12 @@
   <main class="govuk-main-wrapper" role="mail">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        <h1 class="govuk-heading-l">Get email alerts</h1>
-        <p class="govuk-body">You are signing up for email alerts about: how to prepare for a no deal Brexit.
-        <p class="govuk-body">
-          This will include: Information about Brexit including what you and your business can do to prepare.
```

Comment:
Moved this to the itnernationalisation YAML for consistency.
hard coded content makes me sad.